entry/src/main/java/kr/pe/burt/ohos/lib/ohos/channel/app/MainAbility.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (C) 2020-21 Application Library Engineering Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kr.pe.burt.ohos.lib.ohos.channel.app;
+
+import ohos.aafwk.ability.fraction.FractionAbility;
+import ohos.aafwk.content.Intent;
+import ohos.agp.components.Text;
+import ohos.agp.utils.Color;
+import ohos.eventhandler.InnerEvent;
+import ohos.hiviewdfx.HiLog;
+import ohos.hiviewdfx.HiLogLabel;
+import ohos.multimodalinput.event.TouchEvent;
+import kr.pe.burt.ohos.lib.ohoschannel.OhosChannel;
+import kr.pe.burt.ohos.lib.ohoschannel.Timer;
+
+/**
+ * MainAbility Class.
+ */
+public class MainAbility extends FractionAbility {
+    public static final HiLogLabel HI_LOG_LABEL = new HiLogLabel(0, 0, "FractionAbility");
+    static final int PING = 0;
+    static final int PONG = 1;
```

Comment:
Clearly define the scope of the variables.

Reply:
Done